shared/middleware/helpers/auth_helpers.rb

```diff
@@ -29,6 +30,14 @@ def under_13?
   age < 13
 end
 
+# Returns the sharing_disabled property of a user with a given user_id
+def get_user_sharing_disabled(user_id)
+  user_properties = DASHBOARD_DB[:users].select(:properties).first(id: user_id)
```

Comment:
@caleybrock it looks like what you need here to avoid the HB error is `DASHBOARD_DB[:users].select(:properties).first(id: 1)[:properties]`